bindata/network/ovn-kubernetes/alert-rules.yaml

```diff
@@ -15,9 +15,10 @@ spec:
     - alert: NodeWithoutOVNKubeNodePodRunning
       annotations:
         message: |
-          All nodes should be running an ovnkube-node pod, {{"{{"}} $labels.node {{"}}"}} is not.
+          All nodes except Windows nodes, should be running an ovnkube-node pod, {{"{{"}} $labels.node {{"}}"}} is not.
       expr: |
-        (kube_node_info unless on(node) kube_pod_info{namespace="openshift-ovn-kubernetes",  pod=~"ovnkube-node.*"}) > 0
+        (kube_node_info unless on(node) (kube_pod_info{namespace="openshift-ovn-kubernetes",pod=~"ovnkube-node.*"}
+        or kube_node_labels{label_kubernetes_io_os="windows"})) > 0
```

Comment:
Hardcoding is OK for now, but I'd really like to see the label name + value templated into this file and ovnkube-[master|node].yaml by the CNO using a single source. OTherwise we're likely to change one place in the future, and not change this place.